OADP-6644: S3 support statement (#98387)
openshift/openshift-docs: modules/oadp-s3-compatible-backup-storage-providers.adoc

[id="oadp-s3-compatible-backup-storage-providers_{context}"]
= AWS S3 compatible backup storage providers

> OADP is compatible with many object storage providers for use with different backup and snapshot operations. Several object storage providers are fully supported, several are unsupported but known to work, and some have known limitations.

Author: ```suggestion
{oadp-short} is compatible with many object storage providers for use with different backup and snapshot operations. Several object storage providers are fully supported, several are unsupported but known to work, and some have known limitations.
```

[NOTE]
====
Red Hat will provide support for {oadp-short} on any S3-compatible storage, but support will stop if the S3 endpoint is determined to be the root cause of an issue.
====

[id="oadp-certified-backup-storage-providers_{context}"]
== Certified backup storage providers

> The following AWS S3 compatible object storage providers are fully supported by OADP through the AWS plugin for use as backup storage locations:

Author: ```suggestion
The following AWS S3 compatible object storage providers are fully supported by {oadp-short} through the AWS plugin for use as backup storage locations:
```

[id="oadp-s3-compatible-backup-storage-providers_{context}"]
= AWS S3 compatible backup storage providers

> {oadp-short} is compatible with many object storage providers for use with different backup and snapshot operations. Several object storage providers are fully supported, several are unsupported but known to work, and some have known limitations.

Author: ```suggestion
{oadp-short} will work with many S3-compatible object storage providers. Several object storage providers are certified and tested with every release of {oadp-short}. Various S3 providers are known to work with {oadp-short} but are not specifically tested and certified. These providers will be supported on a best-effort basis. Additionally, there are a few S3 object storage providers with known issues and limitations that are listed in this documentation.
```
